projects/public-search/src/app/routes/documents-route.service.ts

```diff
@@ -50,10 +47,10 @@ export class DocumentsRouteService extends BaseRoute implements ResourceRouteInt
   }
 
   /**
-   * Resource name of route
+   * Resource name of routes
    */
-  getName() {
-    return this.routeName;
+  getResources(): string[] {
+    return ['documents', 'persons'];
```

Comment:
Why 2 resources names are used into the `documents-route.service` ?

Reply:
There is one resource per tab in this case.